.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,58 @@
+name: Bug report
+description: Create a report to help us improve
+labels: Bug
+
+body:
+- type: textarea
+  attributes:
+    label: Describe the bug
+    placeholder: A clear and concise description of what the bug is.
```

Comment:
I was thinking that we may encourage users to add screenshots here (so we don't need to have separate `screenshots` field. As it is usually better to have inline image in text describing the issue.

What do you think?

Also it would be great to let people know here to use English for their reports and screenshots.
I think most of us maintainers here use English as language and many times it is hard to even recognize what is going on when screenshot with Cyrillic is provided.